src/components/RegisterForm.test.tsx

```diff
@@ -0,0 +1,205 @@
+import { render, screen, within, fireEvent } from '@testing-library/react'
+import RegisterForm from './RegisterForm'
+import { User } from '../entities/User'
+
+const exampleUser: User.Request = {
+    password: 'Password123',
+    repeatPassword: 'Password123',
+    name: 'Meier',
+    firstname: 'Max',
+    email: 'example@mail.com',
+    telephone: '+41 44 444 44 44',
+    place: 'ExamplePlace',
+    street: 'Examplestreet 11',
+    zip: '2222',
+}
+
+describe('<RegisterForm>', () => {
+    test('show input correctly', () => {
```

Comment:
Instead of writing a test for every input, you could have an array which specifies all inputs like: 
```ts
const inputs = [
  { testId: 'name-input', label: 'Name', value: exampleUser.name },
  { testId: 'firstname-input', label: 'Vorname', value: exampleUser.firstname },
  ...
]
```
Then you could have just one test which does sth like: 
```ts
render(...)
inputs.forEach(input => {
    const label = getLabelOfInput(input.testId) 
    expect ...
})
```

Reply:
improved tests